--- /usr/share/mozilla/extensions/{ec8030f7-c20a-464f-9b0e-13a3a9e97384}/uBlock0@raymondhill.net/js/background.js	2023-01-23 21:02:37.000000000 +0000
+++ /usr/share/mozilla/extensions/{ec8030f7-c20a-464f-9b0e-13a3a9e97384}/uBlock0@raymondhill.net/js/background.js	2023-01-26 11:42:12.811555615 +0000
@@ -91,7 +91,7 @@
 const userSettingsDefault = {
     advancedUserEnabled: false,
     alwaysDetachLogger: true,
-    autoUpdate: true,
+    autoUpdate: false,
     cloudStorageEnabled: false,
     cnameUncloakEnabled: true,
     collapseBlocked: true,
