--- a/usr/share/mozilla/extensions/{ec8030f7-c20a-464f-9b0e-13a3a9e97384}/uBlock0@raymondhill.net/js/background.js	2021-08-18 20:56:50.000000000 +0200
+++ b/usr/share/mozilla/extensions/{ec8030f7-c20a-464f-9b0e-13a3a9e97384}/uBlock0@raymondhill.net/js/background.js	2021-09-09 16:14:33.405574913 +0200
@@ -84,7 +84,7 @@
     const userSettingsDefault = {
         advancedUserEnabled: false,
         alwaysDetachLogger: true,
-        autoUpdate: true,
+        autoUpdate: false,
         cloudStorageEnabled: false,
         cnameUncloakEnabled: true,
         collapseBlocked: true,
