--- /usr/share/mozilla/extensions/{ec8030f7-c20a-464f-9b0e-13a3a9e97384}/uBlock0@raymondhill.net/js/background.js	2021-11-26 23:10:11.000000000 +0100
+++ /usr/share/mozilla/extensions/{ec8030f7-c20a-464f-9b0e-13a3a9e97384}/uBlock0@raymondhill.net/js/background.js	2021-11-29 12:26:50.129438199 +0100
@@ -90,7 +90,7 @@
 const userSettingsDefault = {
     advancedUserEnabled: false,
     alwaysDetachLogger: true,
-    autoUpdate: true,
+    autoUpdate: false,
     cloudStorageEnabled: false,
     cnameUncloakEnabled: true,
     collapseBlocked: true,
