--- a/etc/apparmor.d/usr.sbin.cupsd       2021-10-13 15:28:55.619014891 +0000
+++ b/etc/apparmor.d/usr.sbin.cupsd       2021-10-13 15:32:08.399820622 +0000
@@ -4,6 +4,8 @@
 
 #include <tunables/global>
 
+@{etccups}=/{etc/cups,live/persistence/TailsData_unlocked/cups-configuration}
+
 /usr/sbin/cupsd flags=(attach_disconnected) {
   #include <abstractions/base>
   #include <abstractions/bash>
@@ -63,9 +65,9 @@
   /dev/bus/usb/ r,
   /dev/bus/usb/** rw,
   /dev/parport* rw,
-  /etc/cups/ rw,
-  /etc/cups/** rw,
-  /etc/cups/interfaces/* ixrw,
+  @{etccups}/ rw,
+  @{etccups}/** rw,
+  @{etccups}/interfaces/* ixrw,
   /etc/foomatic/* r,
   /etc/gai.conf r,
   /etc/papersize r,
@@ -80,9 +82,9 @@
   @{PROC}/*/auxv r,
   @{PROC}/sys/crypto/** r,
   /sys/** r,
-  /usr/bin/* ixr,
+  /usr/bin/{[^h],h[^p],hp[^i],hpi[^j],hpij[^s]}* ixr,
   /usr/sbin/* ixr,
-  /{usr/,}bin/* ixr,
+  /{usr/,}bin/{[^h],h[^p],hp[^i],hpi[^j],hpij[^s]}* ixr,
   /{usr/,}sbin/* ixr,
   /usr/lib/** rm,
 
@@ -108,7 +111,15 @@
 
   # third party backends get no restrictions as they often need high
   # privileges and this is beyond our control
-  /usr/lib/cups/backend/* Cx -> third_party,
+  # Due to Tails#9963 we hard-code the third-party backends we ship:
+  # /usr/lib/cups/backend/* Cx -> third_party,
+  /usr/lib/cups/backend/beh Cx -> third_party,
+  /usr/lib/cups/backend/cups-brf Cx -> third_party,
+  /usr/lib/cups/backend/driverless Cx -> third_party,
+  /usr/lib/cups/backend/driverless-fax Cx -> third_party,
+  /usr/lib/cups/backend/gutenprint5[0-9]+usb Cx -> third_party,
+  /usr/lib/cups/backend/hp Cx -> third_party,
+  /usr/lib/cups/backend/hpfax Cx -> third_party,
 
   /usr/lib/cups/cgi-bin/* ixr,
   /usr/lib/cups/daemon/* ixr,
@@ -133,6 +143,7 @@
   /var/cache/cups/** rwk,
   /var/log/cups/ rw,
   /var/log/cups/* rw,
+  /var/spool/ rwk,
   /var/spool/cups/ rw,
   /var/spool/cups/** rw,
 
@@ -147,7 +158,7 @@
   /etc/krb5.conf r,
   deny /etc/krb5.conf w,
   /etc/krb5.keytab rk,
-  /etc/cups/krb5.keytab rwk,
+  @{etccups}/krb5.keytab rwk,
   /tmp/krb5cc* k,
 
   # likewise authentication
@@ -203,8 +214,8 @@
   /{usr/,}bin/bash ixr,
   /{usr/,}bin/cp ixr,
   /etc/papersize r,
-  /etc/cups/cups-pdf.conf r,
-  /etc/cups/ppd/*.ppd r,
+  /@{etccups}/cups-pdf.conf r,
+  /@{etccups}/ppd/*.ppd r,
   /usr/bin/gs ixr,
   /usr/lib/cups/backend/cups-pdf mr,
   /usr/lib/ghostscript/** mr,
