--- a/etc/xdg/menus/gnome-applications.menu	2020-03-11 14:08:10.000000000 +0000
+++ b/etc/xdg/menus/gnome-applications.menu	2021-10-14 11:33:07.148425596 +0000
@@ -35,7 +35,6 @@
            Please keep the list alphabetically sorted! -->
       <Filename>eog.desktop</Filename>
       <Filename>gucharmap.desktop</Filename>
-      <Filename>org.gnome.Calculator.desktop</Filename>
       <Filename>org.gnome.DejaDup.desktop</Filename>
       <Filename>org.gnome.Dictionary.desktop</Filename>
       <Filename>org.gnome.DiskUtility.desktop</Filename>
@@ -299,7 +298,6 @@
       <Filename>gnome-system-log.desktop</Filename>
       <Filename>gnome-system-monitor.desktop</Filename>
       <Filename>org.gnome.baobab.desktop</Filename>
-      <Filename>org.gnome.Terminal.desktop</Filename>
       <Filename>org.gnome.Usage.desktop</Filename>
     </Exclude>
 
@@ -421,7 +419,6 @@
       <Filename>gucharmap.desktop</Filename>
       <Filename>nm-connection-editor.desktop</Filename>
       <Filename>org.gnome.baobab.desktop</Filename>
-      <Filename>org.gnome.Calculator.desktop</Filename>
       <Filename>org.gnome.DejaDup.desktop</Filename>
       <Filename>org.gnome.Dictionary.desktop</Filename>
       <Filename>org.gnome.DiskUtility.desktop</Filename>
@@ -430,7 +427,6 @@
       <Filename>org.gnome.fonts.desktop</Filename>
       <Filename>org.gnome.Screenshot.desktop</Filename>
       <Filename>org.gnome.seahorse.Application.desktop</Filename>
-      <Filename>org.gnome.Terminal.desktop</Filename>
       <Filename>org.gnome.tweaks.desktop</Filename>
       <Filename>org.gnome.Usage.desktop</Filename>
       <Filename>simple-scan.desktop</Filename>
